core/arch/arm/plat-poplar/conf.mk

```diff
@@ -25,7 +25,7 @@ ifeq ($(CFG_PL061),y)
 core-platform-cppflags		+= -DPLAT_PL061_MAX_GPIOS=104
 endif
 
-CFG_SECURE_DATA_PATH ?= y
+CFG_SECURE_DATA_PATH ?= n
```

Comment:
Please just remove the line. We already have `CFG_SECURE_DATA_PATH ?= n` in `mk/config.mk`.

Reply:
done, pushed new version with tag applied. thanks